lib/matplotlib/path.py

```diff
@@ -667,13 +667,27 @@ def intersects_bbox(self, bbox, filled=True):
 
     def interpolated(self, steps):
         """
-        Return a new path resampled to length N x *steps*.
+        Return a new path with each `LINETO` resampled to *steps* `LINETO`'s.
```

Comment:
```suggestion
        Return a new path with each segment is divided into *steps* parts.
```
Note that "segment" is the term for one "step" (c.f. https://matplotlib.org/devdocs/api/path_api.html#matplotlib.path.Path.iter_segments)

Edit: I've just read one maybe "each LINETO step ..."